src/components/Map/FcsMap/FcsFoodSecurityAccordion.tsx

```diff
@@ -0,0 +1,103 @@
+import { Spacer } from '@nextui-org/react';
+
+import FcsAccordionProps from '@/domain/props/FcsAccordionProps';
+import { FcsAccordionOperations } from '@/operations/map/FcsAccordionOperations';
+import { cardsWrapperClass } from '@/utils/primitives';
+
+import { ReactComponent as FoodConsumption } from '../../../../public/Images/FoodConsumption.svg';
+import { ReactComponent as Population } from '../../../../public/Images/Population.svg';
+import CustomCard from '../../Cards/Card';
+import { LineChart } from '../../Charts/LineChart';
+import CustomInfoCircle from '../../CustomInfoCircle/CustomInfoCircle';
+
+export default function FoodSecurityAccordion({ countryData }: FcsAccordionProps) {
```

Comment:
this is not a good idea. You are styling this as a react component and then returning accordion items. You should create a function in operations to return accordion items and then use it in FcsAccordion